template/package.json

```diff
@@ -34,5 +35,10 @@
   },
   "devDependencies": {
     "parcel-plugin-nuke-dist": "^1.0.0"
+  },
+  "xo": {
+    "extends": [
+      "tram-one"
+    ]
```

Comment:
I believe this will fail without the eslint-config-tram-one dependency... please include that in the template as well. You should be able to verify by running `npx xo` in the generated project.

Reply:
Check the latest commit, I added that package.
Though `npx xo` reports 48 errors in the example built with tram-one.

Reply:
That's fine for now, I can make a separate issue to update the existing template.